### ISSUE TITLE
Add option to `put` straight to NRS URL

### ISSUE THREAD

**Issue by S-Coyle** [maidsafe/safe-api#537]
Some users may not care about FilesContainer xorurl addresses, their common use case being to publish sites with an NRS. At the moment this is a 2 step process, perhaps we can have the option to create an NRS from the initial `put`?

## Current Behavior
1. `safe files put ./abcde/ --recursive`
2. `safe nrs create abcde --link "safe://xxxxxxxx?v=0"`

**Comment by dan-da** [comment]
how do you envision this command?   maybe something like:

```
safe files put ./abcde/ --recursive --create-nrs=abcde
```

**Comment by S-Coyle** [comment]
@dan-da Yes, your example above would be how I'd imagine it would be be structured.

I had a look through the user guide to see if there was a mention of another `create` option to see how it was structured, but I don't see one we need to use as a template, so the above works for me.
